trk/src/Navigation/AppNavigator.js

```diff
@@ -708,7 +713,7 @@ function AppTabs() {
           },
         }}
 
-      />)}
+      />
```

Comment:
So you added back the collection tab for routesetters?

Reply:
Oh, yes. Sorry, forgot to remove it. good catch.